src/utils.ts

```diff
@@ -1,5 +1,6 @@
 import { execSync } from 'child_process';
 import fs from 'fs';
+import toml from '@iarna/toml';
 import { getFullnodeUrl } from '@mysten/sui/client';
 
 export function getModuleNameFromSymbol(symbol: string) {
```

Comment:
this method seems unnecessarily complicated as well, can you take a look? where is this used?

Reply:
seems like it doesn't use anywhere. do you really need it?

Reply:
don't see it used in deployments repo as well, let's remove it